### PR TITLE
docs: added meeting minutes

### PR DESCRIPTION
**Pull request by UlisesGascon**

### CODE REVIEW

File: docs/meetings/2023-02-23.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- Next version (potentially next week - as beta) will auto-scope the orgs repos and won't require any effort to maintain it.
+- Once new version is released, Ulises will create a PR with the implementation for One Beyond org in the maintainers repo.
+- It will potentially require some extra PRs to proactively notificate the new scores to the OpenSSF API.
+
+
+## Q&A, Other
+
+- Potential plan to Move systemic extension a mono-repo
+  - We don’t have a full agreement on the need to move for mono-repo approach in the group
+  - We will need to research about this topic to consolidate a proper discussion (changelogs, git history migration, etc...)
```
Comment (inigomarquinez) [inline docs/meetings/2023-02-23.md:57]: @UlisesGascon , do you find it useful to add [this link](https://medium.com/@filipenevola/how-to-migrate-to-mono-repository-without-losing-any-git-history-7a4d80aa7de2) that explains how to migrate to monorepo and keep the git history?

Comment (UlisesGascon) [reply docs/meetings/2023-02-23.md:57]: I will say better [here](https://github.com/onebeyond/maintainers/issues/52) to centralize the discussion :)

Comment (inigomarquinez) [reply docs/meetings/2023-02-23.md:57]: Agree :)

**Review by inigomarquinez — APPROVED** [review]
LGTM!

**Pull request merged**